2D Simulation/game.js

```diff
@@ -160,6 +140,27 @@ socket.on('update_flip_block', function (block_id) {
     flipBlock(block_id, null, currentConfig);
 });
 
+socket.on('indicate_impossible_move', function(move) {
```

Comment:
If you're adding new Socket.io signals, you need to update the sequence diagrams in the docs.

Reply:
If I am reading them correctly they are already out of date, I think that the AI mode should be added in a separate PR.

Reply:
There is the AI mode sequence diagram in #42. I'll quickly pull that into its own PR and get that pushed up.